src/test/resources/xml/IE015SimplifiedPrelodgedDepartureDeclarationFull.xml

```diff
@@ -8,7 +8,7 @@
         <LRN>38VYQTYFU3T0KUTUM3</LRN>
         <declarationType>Pbg</declarationType>
         <additionalDeclarationType>D</additionalDeclarationType>
-        <security>8</security>
+        <security>1</security>
```

Comment:
Are we not testing scenarios where the security is 0?

Reply:
i can create 1 new scenario, but as we dont have form previous sections not added that one

Reply:
added this pls check